Doppler.ContactPolicies.Api.Test/UpdateContactPoliciesSettingsTest.cs

```diff
@@ -34,29 +32,27 @@ public UpdateContactPoliciesSettingsTest(WebApplicationFactory<Startup> factory)
         }
 
         [Theory]
-        [InlineData("test1@test.com", CONTACT_POLICIES_SETTINGS, TOKEN_ACCOUNT_123_TEST1_AT_TEST_DOT_COM_EXPIRE_20330518,
+        [InlineData(CONTACT_POLICIES_SETTINGS_REQUEST_BODY_STUB, "test1@test.com", TOKEN_ACCOUNT_123_TEST1_AT_TEST_DOT_COM_EXPIRE_20330518,
```

Comment:
Do we have other scenarios for this test? What do you think about using `Fact` instead of `Theory`?